scheduler/src/cook/mesos/api.clj

```diff
@@ -1841,6 +1841,13 @@
                              jobs)))
       [true {::error (str "Increment would exceed the maximum retry limit of " retry-limit)}]
 
+      (and retries (let [db (d/db conn)]
+                     (some (fn [job]
+                             (> (d/invoke db :job/attempts-consumed db (d/entity db [:job/uuid job]))
+                                retries))
```

Comment:
Should this be `>=`?

Reply:
It's legal to set it to the current attempts consumed, you just can't go lower.